octobot_trading/exchanges/exchange_simulator.py

```diff
@@ -68,7 +69,10 @@ def __init__(self, config, exchange_type, exchange_manager, backtesting_data_fil
     async def create_backtesting_exchange_producers(self):
         for importer in self.exchange_importers:
             for updater in UNAUTHENTICATED_UPDATER_SIMULATOR_PRODUCERS:
-                await updater(get_trading_chan(updater.CHANNEL_NAME, self.exchange_manager.id), importer).run()
+                if updater == MarkPriceUpdater:
```

Comment:
Oh ok we discussed this change in #88 but it was from this PR... How should we proceed ? You want to merge this and modify this line in #88 or modify this PR that would be cleaner ?

Reply:
I will remove this in this PR